src/index.js

```diff
@@ -62,10 +60,10 @@ $(document).ready(function() {
       if (mocha) {
         clearInterval(mochaCheck);
         mocha.setup('bdd');
-        const testDiv = document.createElement('div');
+        var testDiv = document.createElement('div');
         testDiv.style.position = 'inherit';
-        testDiv.innerHTML = testSuiteSkeleton;
-        document.body.appendChild(testDiv);
+        testDiv.innerHTML = testSuiteSkeleton.default;
```

Comment:
This can be changed to the simpler `testDiv.innerHTML = testSuiteSkeleton` if we use the CommonJS export option (see my other comment).